### PR TITLE
Add Eval Command

### PR DESCRIPTION
**Pull request by officialpiyush**
# What's this PR About?

* This PR adds an eval command to the boiler directory.

## Linting?

* This PR Is Linted as per the rules in [Eslint rules](https://github.com/alex-taxiera/eris-boiler/blob/master/.eslintrc.yml) File
image

### CODE REVIEW

File: boiler/commands/Admin/eval.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    middleware: adminOnly,
+    parameters: [ 'code' ]
+  },
+  run: async ({ bot, msg, params }) => {
+    let evaled
+    try {
+      const stopwatchStart = process.hrtime()
+      evaled = eval(params.join(' ')) // eslint-disable-line no-eval
+      if (evaled instanceof Promise) {
+        evaled = await evaled
```
Comment (alex-taxiera) [inline boiler/commands/Admin/eval.js]: can't you just do `await eval()` regardless of whether it is a promise or not?

File: boiler/commands/Admin/eval.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      const stopwatchEnd = process.hrtime(stopwatchStart)
+
+      let response = '' // Woow, an empty string? see below
+
+      response += `**Output:**\n\`\`\`js\n${clean(
+        inspect(evaled, { depth: 0 }),
+        bot.token
+      )}\`\`\``
+      response += `\n\n**Type:**\`\`\`${typeof evaled}\`\`\``
+      response += `\n\n\n⏱️ \`${(stopwatchEnd[0] * 1e9 + stopwatchEnd[1]) /
```
Comment (alex-taxiera) [inline boiler/commands/Admin/eval.js]: why is response defined and then added to when there are no conditional adds? Could this not be accomplished by `const response = '...'`?

File: boiler/commands/Admin/eval.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        1e6}ms\``
+
+      if (response.length > 0) {
+        // To Check if response has something to send xd
+        return response
```
Comment (alex-taxiera) [inline boiler/commands/Admin/eval.js]: if response is empty we should probably return a message stating that it had no return value.

**Comment by officialpiyush** [comment]
I'll look into it soon hopefully, I don't have access to my pc rn

**Comment by alex-taxiera** [comment]
closing due to inactivity on outdated code

**Pull request closed without merging**